docs/.vitepress/theme/components/Home.vue

```diff
@@ -192,7 +249,7 @@ const subscribeToNewsletter = async () => {
     </li>
     <li>
       <div class="text-off-white">🏭 Tags and versions</div>
-      <p class="p2 mb-4 text-gray-06">Each ModelKit is tagged so everyone knows which dataset and model work together.</p>
+      <p class="p2 mb-4 text-gray-06">Each ModelKit is tagged, versioned, and timestamped so everyone knows which dataset and model work together.</p>
```

Comment:
Small note: on the CLI side, we're not doing timestamping (in order to ensure digests are reproducible)

Reply:
@javisperez updated g-doc